libs/backend-apisix/e2e/sync-and-dump-1.e2e-spec.ts

```diff
@@ -424,11 +424,11 @@ describe('Sync and Dump - 1', () => {
     const plugin1Name = 'prometheus';
     const plugin1 = {
       prefer_name: true,
-    } as ADCSDK.GlobalRule;
+    } as unknown as ADCSDK.GlobalRule;
```

Comment:
It should be `ADCSDK.Plugin`. The old code was incorrect, and using a forced type cast was also incorrect.